bin/crew

```diff
@@ -249,6 +249,10 @@ end
 def update
   abort "'crew update' is used to update crew itself. Use 'crew upgrade <package1> [<package2> ...]' to update specific packages.".orange if @pkg_name
 
+  # The following is used in fixup.rb to determine if crew update needs to
+  # be run again.
+  @crew_const_git_commit = `git -C #{CREW_LIB_PATH} log -n1 --oneline #{CREW_LIB_PATH}/lib/const.rb`.split.first
```

Comment:
Why are we assigning a variable (and calling it a constant) that is not even used anywhere?

Reply:
It's used in fixup.rb, as per the comment in the changed lines.

Reply:
I read that.  We're not using it in crew though so why not assign it there instead?

Reply:
It is assigned in crew and also in fixup if not already defined.

Reply:
So what is confusing me is why are we defining it here if there is no logic to use it?  Why not just define in fixup.rb since that is where it's needed?